_data/navbars/reference.yml

```diff
@@ -34,8 +34,6 @@ section:
       path: /reference/calicoctl/ipam/release
     - title: show
       path: /reference/calicoctl/ipam/show
-    - title: configure
-      path: /reference/calicoctl/ipam/configure
```

Comment:
https://docs.projectcalico.org/master/reference/calicoctl/ipam/configure appears to still be a valid link.  Why has it been removed in this PR?  (Not saying removing it might not be valid, just that it's unrelated to this PR's supposed purpose.)?

Reply:
great catch! I have no idea how I managed to remove it since its not even related to what I was working on. I think it was a mistake on my part.